README.md

```diff
@@ -9,19 +9,28 @@
 
 ### Install Prerequisites
 #### Install Git
- * If you want to use Git in the command line, run `sudo apt install git`
+ * If you want to use Git in the command line, run `sudo apt install git` on Ubuntu, on Windows you can download [Git for Windows](https://git-scm.com/) or use `choco install git` if you have Chocolatey installed.
  * If you're less terminal-inclined, you can install a Git GUI wrapper. [There's a nice list here.](https://git-scm.com/downloads/guis/)
     * I would personally recommend [GitKraken](https://www.gitkraken.com/). The free version has everyhting you need, (although their conflict resolver for Pro is nice,) and you can also get Pro free with [GitHub Education](https://education.github.com/)!
+#### Install CMake
+ * Debian-based (Ubuntu) run `sudo apt install cmake`
+ * Fedora/RHEL-based `sudo dnf install cmake`
+ * Windows Download [CMake](https://cmake.org/download/) or run `choco install cmake` if you have Chocolatey installed.
+#### Install vcpkg
+ * Vcpkg is a cross platform library manager, you can install it by reading the instructions on this [page](https://github.com/microsoft/vcpkg)
+ * This program is required to build echo-bot under Windows.
 #### <sup>AND</sup> Install C++
  1. Install a C++ compiler
-    * the typical standard is `gcc`, you can install it with `sudo apt install gcc`
-    * `clang` is an alternative that I feel has better error statements, you can install it with `sudo apt install clang`
- 2. Install CMake with `sudo apt install cmake`
+    * the typical standard on Linux is `gcc`, you can install it on Ubuntu with `sudo apt install gcc`
+    * `clang` is an alternative that I feel has better error statements, you can install it on Ubuntu with `sudo apt install clang`
+    * The typical compiler on Windows is [Microsoft Visual C++](https://www.visualstudio.com/)
+    * [MinGW](https://www.msys2.org/) or [Cygwin](https://www.cygwin.com/) are valid alternatives if you like GCC running on Windows, oh there's also [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) as well ^^.
```

Comment:
Maybe clarify a tiny bit what WSL does? Something like "WSL, which allows you to use a Linux development environment on Windows"